qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -264,6 +264,67 @@ def test_get_to_console(self):
         log.info("o_hash:{}".format(o_hash))
         assert(s_hash == o_hash)
 
+    def test_snap(self):
+        """
+        Test that snapshot creation and deletion work
+        """
+        sd = self.fs.get_config('client_snapdir')
+        sdn = "data_dir/{}/snap1".format(sd)
+
+        # create a data dir and dump some files into it
+        self._cephfs_shell("mkdir data_dir")
+        s = 'A' * 10240
+        o = self._cephfs_shell("put - data_dir/data_a", stdin=s)
+        s = 'B' * 10240
+        o = self._cephfs_shell("put - data_dir/data_b", stdin=s)
+        s = 'C' * 10240
+        o = self._cephfs_shell("put - data_dir/data_c", stdin=s)
+        s = 'D' * 10240
+        o = self._cephfs_shell("put - data_dir/data_d", stdin=s)
+        s = 'E' * 10240
+        o = self._cephfs_shell("put - data_dir/data_e", stdin=s)
+
+        o = self._cephfs_shell("ls -l /data_dir")
+        log.info("cephfs-shell output:\n{}".format(o))
+
+        # create the snapshot - must pass
```

Comment:
It would be nicer if we can move each testcase into different methods.

Reply:
since the test methods run sequence cannot be guaranteed its difficult to split the test into different methods